packages/vitepress-plugin-og-image-generator/src/index.ts

```diff
@@ -0,0 +1,100 @@
+import { fileURLToPath } from 'node:url'
+import { dirname, relative, resolve, sep } from 'node:path'
+import fs from 'fs-extra'
+import fg from 'fast-glob'
+import { Resvg } from '@resvg/resvg-js'
+import type { Config } from './types/config'
+import type { ArticleTree } from './types/metadata'
+import { removeEmoji } from './utils'
+
+export default async function run(config: Config) {
```

Comment:
Where would it be imported? Can you install this plugin into `@nolebase/integrations-docs` located under `/docs` to configure and make try?

Reply:
Sorry, but where is the `@nolebase/integrations-docs`?